src/Student9/Ex3/Main.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react'
-import './Main.css'
-
+import { Tweets } from './Tweets'
+import './Main.scss'
 export const Main = () => {
   return (
-    <div>
-        Hello I'm student id: 3.
+    <div className="Tweets">
+      <div>Tweet Storm</div>
+      <Tweets message="It's payday!!!"  color="green" />
```

Comment:
For Ex. 3, we need to create a Tweet component.   The Tweets Component will contain 3 Tweet components (like in the screen shot above) .  This structure is important.

Please make a Tweet component.